packages/sanity/src/core/store/_legacy/user/__tests__/userStore.test.ts

```diff
@@ -78,15 +78,15 @@ describe('userStore', () => {
       ])
     })
 
-    it('does not refetch current user if passed when constructing user store, normalizes', async () => {
+    it('refetches current user if passed when constructing user store, as currentUser does not contain sanityUserId (global id)', async () => {
```

Comment:
If we agree on the change in userStore (se below), I will rewrite this test to show that there is refetching going on. This is just a quickfix to get it passing for now.

Reply:
Reverted.